templates/resources/amplify_branch.tmpl

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
the file name should be called amplify_branch.md.tmpl

Reply:
whoops - fixed typo in file name and ran `make docs` again. Should be corrected now